aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -133,6 +133,28 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 				},
 			},
 
+			"source_configuration": {
```

Comment:
I would keep "kinesis_source_configuration", since we can have other sources potentially in the future. Thoughts?

Reply:
Yeah, I agree, I'll update